api/envoy/extensions/load_balancing_policies/client_side_weighted_round_robin/v3/client_side_weighted_round_robin.proto

```diff
@@ -68,4 +69,12 @@ message ClientSideWeightedRoundRobin {
   // calculated as eps/qps. Configuration is rejected if this value is negative.
   // Default is 1.0.
   google.protobuf.FloatValue error_utilization_penalty = 6 [(validate.rules).float = {gte: 0.0}];
+
+  // By default, endpoint weight is computed based on the
+  // "application_utilization" field reported by the endpoint.  If that field is not set, then utilization
+  // will instead be computed by taking the max of the values of the metric
+  // names specified here.  If none of the specified metrics are present, then
+  // "cpu_utilization" is
+  // used instead.
+  repeated string utilization_from_metric_names = 7;
```

Comment:
can we specify in the comments what the possible values are?  if they are a fixed known set, would an enum be more suitable?

also, for naming, i wonder if `metrics_for_computing_utilization` makes it a bit more clear?

Reply:
I think the field name should include the string `metric_names`, since that's ultimately what this field contains.  I think it should *not* say `named_metrics`, since that implies a correllation specifically with the `named_metrics` ORCA field, whereas (as per my comment above), I think the metrics names here should be able to be from any ORCA field.

I think the current name is fine, but I'd also be okay with something like `metric_names_for_computing_utilization`.

Reply:
I've renamed this field into `metric_names_for_computing_utilization`.